client/components/theme-collection/index.tsx

```diff
@@ -0,0 +1,126 @@
+import { Button } from '@wordpress/components';
+import { Icon, chevronLeft, chevronRight } from '@wordpress/icons';
+import { ReactElement, useEffect, useRef } from 'react';
+import './style.scss';
+import { Swiper as SwiperType } from 'swiper/types';
+import { ThemeBlock } from 'calypso/components/themes-list';
+
+interface ThemeCollectionProps {
+	heading: string;
+	subheading: ReactElement;
+	themes: Array< never >;
+	collectionSlug: string;
+	getScreenshotUrl: ( themeId: string ) => string;
+	siteId: string | null;
+	getButtonOptions: () => void;
+	getActionLabel: () => string;
+	isActive: () => boolean;
+	getPrice: () => string;
+	isInstalling: () => boolean;
+}
+
+export default function ThemeCollection( {
+	collectionSlug,
+	heading,
+	subheading,
+	themes,
+	getScreenshotUrl,
+	siteId,
+	getButtonOptions,
+	getActionLabel,
+	isActive,
+	getPrice,
+	isInstalling,
+}: ThemeCollectionProps ): ReactElement {
+	const swiperInstance = useRef< SwiperType | null >( null );
+	useEffect( () => {
+		if ( themes !== null && themes.length > 0 ) {
+			const el = document.querySelector( '.swiper-container' ) as HTMLElement;
+			if ( el ) {
+				Promise.all( [
```

Comment:
We have to import the swiper package dynamically because it doesn't offer a CommonJS version. Because of this, the SSR build will fail.

From a performance standpoint, I don't think it will cause any issues; we are already loading the ThemeCards dynamically anyway.

Reply:
This should be a code comment really

Reply:
Good point!